packages/lumx-react/src/components/thumbnail/Thumbnail.stories.tsx

```diff
@@ -49,3 +49,44 @@ export const defaultThumbnail = ({ theme }: { theme: Theme }) => {
         />
     );
 };
+
+export const defaultThumbnailWithCustomImgProps = ({ theme }: { theme: Theme }) => {
+    const [image, setImage] = React.useState('https://not-found');
+    const onError = () => {
+        setImage('https://i.picsum.photos/id/1001/2400/1400.jpg');
+    };
+
+    return (
+        <Thumbnail
+            align={select<Alignment>('Alignment', Alignment, Alignment.left, 'Options')}
+            aspectRatio={select<AspectRatio>('Aspect ratio', AspectRatio, AspectRatio.square, 'Options')}
+            isCrossOriginEnabled={boolean('Enable CORS', true, 'Options')}
+            crossOrigin={select('CORS', CrossOrigin, CrossOrigin.anonymous, 'Options')}
+            fillHeight={boolean('Fill Height', false, 'Options')}
+            focusPoint={{
+                x: number('focusX', 0, numberKnobOtions, 'Options'),
+                y: number('focusY', 0, numberKnobOtions, 'Options'),
+            }}
+            image={image}
+            size={select(
+                'Size',
+                {
+                    XXS: Size.xxs,
+                    XS: Size.xs,
+                    S: Size.s,
+                    M: Size.m,
+                    L: Size.l,
+                    XL: Size.xl,
+                    XXL: Size.xxl,
+                },
+                Size.xxl,
+                'Options',
+            )}
+            theme={theme}
+            isFollowingWindowSize={boolean('Update on window resize', true, 'Options')}
+            resizeDebounceTime={number('Debounce time after resize', 20, undefined, 'Options')}
+            variant={select<ThumbnailVariant>('Variant', ThumbnailVariant, ThumbnailVariant.squared, 'Options')}
+            imgProps={{ onError }}
```

Comment:
be careful it might not work on IE
https://caniuse.com/#search=img%20onError

Reply:
this is just a story, we will do that check on the product side, not here